src/parser/Parser.Class.spec.ts

```diff
@@ -27,7 +28,8 @@ describe('parser class', () => {
             TokenKind.Public,
             TokenKind.Protected,
             TokenKind.Private,
-            TokenKind.Override
+            TokenKind.Override,
```

Comment:
I'm not sure I agree with preventing these as class properties. Is there a technical reason why we need to prevent them from being properties?